<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(77, 77, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="circularProgressBar_Main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="circularProgressCPU">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.400 rgba(85, 170, 255, 255), stop:0.395 rgba(255, 255, 255, 0));
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circularBg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: rgba(85, 85, 127, 100);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circularContainer">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>25</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 95px;	
	background-color: rgb(58, 58, 102);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>125</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="infoLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelAplicationName">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF; background-color: none;</string>
         </property>
         <property name="text">
          <string>&lt;strong&gt;CPU&lt;/strong&gt; USAGE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPercentageCPU">
         <property name="font">
          <font>
           <family>Roboto Thin</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(115, 185, 255); padding: 0px; background-color: none;</string>
         </property>
         <property name="text">
          <string>&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:50pt;&quot;&gt;60&lt;/span&gt;&lt;span style=&quot; font-size:40pt; vertical-align:super;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelCredits">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(148, 148, 216); background-color: none;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;TEMP: &lt;span style=&quot; color:#ffffff;&quot;&gt;40ยบ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>circularBg</zorder>
    <zorder>circularProgressCPU</zorder>
    <zorder>circularContainer</zorder>
   </widget>
   <widget class="QFrame" name="circularProgressBar_Main_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="circularProgressGPU">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.600 rgba(85, 255, 127, 255), stop:0.595 rgba(255, 255, 255, 0));
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circularBg_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: rgba(85, 85, 127, 100);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circularContainer_2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>25</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 95px;	
	background-color: rgb(58, 58, 102);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>127</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="infoLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelAplicationName_2">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF; background-color: none;</string>
         </property>
         <property name="text">
          <string>&lt;strong&gt;GPU&lt;/strong&gt; USAGE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPercentageGPU">
         <property name="font">
          <font>
           <family>Roboto Thin</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(115, 255, 171); padding: 0px; background-color: none;</string>
         </property>
         <property name="text">
          <string>&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:50pt;&quot;&gt;40&lt;/span&gt;&lt;span style=&quot; font-size:40pt; vertical-align:super;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelCredits_2">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(148, 148, 216); background-color: none;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;TEMP: &lt;span style=&quot; color:#ffffff;&quot;&gt;60ยบ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>circularBg_2</zorder>
    <zorder>circularProgressGPU</zorder>
    <zorder>circularContainer_2</zorder>
   </widget>
   <widget class="QFrame" name="circularProgressBar_Main_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="circularProgressRAM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.750 rgba(255, 0, 127, 255), stop:0.745 rgba(255, 255, 255, 0));
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circularBg_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>220</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: rgba(85, 85, 127, 100);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circularContainer_3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>25</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 95px;	
	background-color: rgb(58, 58, 102);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>125</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="infoLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelAplicationName_3">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF; background-color: none;</string>
         </property>
         <property name="text">
          <string>&lt;strong&gt;RAM&lt;/strong&gt; USAGE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPercentageRAM">
         <property name="font">
          <font>
           <family>Roboto Thin</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 44, 174); padding: 0px; background-color: none;</string>
         </property>
         <property name="text">
          <string>&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:50pt;&quot;&gt;25&lt;/span&gt;&lt;span style=&quot; font-size:40pt; vertical-align:super;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelCredits_3">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(148, 148, 216); background-color: none;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;TEMP: &lt;span style=&quot; color:#ffffff;&quot;&gt;20ยบ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>circularBg_3</zorder>
    <zorder>circularProgressRAM</zorder>
    <zorder>circularContainer_3</zorder>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>4</y>
      <width>641</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 185, 255); background-color: none;</string>
    </property>
    <property name="text">
     <string>This Is My App - Title Bar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>724</x>
      <y>20</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Close</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 0, 0);
}
QPushButton:hover {		
	background-color: rgba(255, 0, 0, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>582</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(220, 220, 220);
background-color: rgb(98, 98, 162);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>&quot;If you control the code, you control the world.&quot;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_maximize">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Maximize</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;	
	background-color: rgb(85, 255, 127);
}
QPushButton:hover {	
	background-color: rgba(85, 255, 127, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimize">
    <property name="geometry">
     <rect>
      <x>697</x>
      <y>20</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Minimize</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 170, 0);
}
QPushButton:hover {	
	background-color: rgba(255, 170, 0, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sliderCPU">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* SLIDERS */
QSlider::groove:horizontal {
    border-radius: 9px;
    height: 18px;
	margin: 0px;
	background-color: rgb(52, 59, 72);
}
QSlider::groove:horizontal:hover {
	background-color: rgb(55, 62, 76);
}
QSlider::handle:horizontal {
    background-color: rgb(85, 170, 255);
    border: none;
    height: 18px;
    width: 18px;
    margin: 0px;
	border-radius: 9px;
}
QSlider::handle:horizontal:hover {
    background-color: rgb(105, 180, 255);
}
QSlider::handle:horizontal:pressed {
    background-color: rgb(65, 130, 195);
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderGPU">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* SLIDERS */
QSlider::groove:horizontal {
    border-radius: 9px;
    height: 18px;
	margin: 0px;
	background-color: rgb(52, 59, 72);
}
QSlider::groove:horizontal:hover {
	background-color: rgb(55, 62, 76);
}
QSlider::handle:horizontal {
    background-color: rgb(85, 255, 127);
    border: none;
    height: 18px;
    width: 18px;
    margin: 0px;
	border-radius: 9px;
}
QSlider::handle:horizontal:hover {
    background-color: rgb(115, 255, 148);
}
QSlider::handle:horizontal:pressed {
    background-color: rgb(65, 195, 95);
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderRAM">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>340</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* SLIDERS */
QSlider::groove:horizontal {
    border-radius: 9px;
    height: 18px;
	margin: 0px;
	background-color: rgb(52, 59, 72);
}
QSlider::groove:horizontal:hover {
	background-color: rgb(55, 62, 76);
}
QSlider::handle:horizontal {
    background-color: rgb(255, 0, 127);
    border: none;
    height: 18px;
    width: 18px;
    margin: 0px;
	border-radius: 9px;
}
QSlider::handle:horizontal:hover {
    background-color: rgb(255, 55, 155);
}
QSlider::handle:horizontal:pressed {
    background-color: rgb(199, 0, 99);
}
</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
